tsconfig.json

```diff
@@ -27,6 +27,6 @@
     }
    
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "src/app/api/cron/notifications/attendance-report/route.tsx", "src/app/(app)/attendance-report/components/MyDocument.js"],
```

Comment:
why we need here this ?

Reply:
Automatically generated , and this line is not necessary src/app/(app)/attendance-report/components/MyDocument.js